# 08/02/2023: Allow newer versions of meson to build this package.
#             Special thanks to Bob Funk who kindly pointed this out.
# Original patch url: https://gitlab.gnome.org/GNOME/gnome-bluetooth/-/commit/755fd758f866d3a3f7ca482942beee749f13a91e.patch            

diff --git a/sendto/meson.build b/sendto/meson.build
index 24a4e394..3e233a03 100644
--- a/sendto/meson.build
+++ b/sendto/meson.build
@@ -22,7 +22,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file (
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
-- 
